### PR TITLE
InputNumber: add step-strictly attribute

### PR DESCRIPTION
**Pull request by luckyCao**
add multiple-of-step attribute
Please make sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md) | [Español](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by island205** [comment]
doc for `multiple-of-step` attribute?

**Comment by luckyCao** [comment]
@island205 review 一下，`multiple-of-step` 这个属性名称没问题 ok 的话，我现在加上

File: examples/docs/en-US/input-number.md [hunk 1]
```diff
@@ -69,6 +69,26 @@ Allows you to define incremental steps.
...
+
+```html
+<template>
+  <el-input-number v-model="num3" :step="2" multiple-of-step></el-input-number>
+</template>
+<script>
+  export default {
+    data() {
+      return {
+        num3: 2
```
Comment (island205) [inline examples/docs/en-US/input-number.md]: 这里可以直接使用 num，这个文档的其他的地方也可以改一下。

File: examples/docs/en-US/input-number.md [hunk 2]
```diff
@@ -69,6 +69,26 @@ Allows you to define incremental steps.
 ```
 :::
 
+### Multiple of Step
+
+:::demo The `multiple-of-step` attribute accepts a `boolean`. if this attribute is `true`, input value can not be anything but multiple of step.
```
Comment (wacky6) [inline examples/docs/en-US/input-number.md]: how about: "input value can only be multiples of `step`."

Comment (luckyCao) [reply examples/docs/en-US/input-number.md]: This is better, thanks

**Comment by ziyoung** [comment]
命名为 `step-strictly`。

File: test/unit/specs/input-number.spec.js [hunk 3]
```diff
@@ -142,6 +142,34 @@ describe('InputNumber', () => {
...
+      data() {
+        return {
+          value: 5
+        };
+      }
+    }, true);
+
+    let input = vm.$el.querySelector('input');
+
+    vm.$nextTick(_ => {
```
Comment (ziyoung) [inline test/unit/specs/input-number.spec.js]: 用 async 函数测试异步过程。

**Pull request merged**